mne/io/brainvision/brainvision.py

```diff
@@ -502,29 +502,30 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                 # highpass relaxed / no filters
                 highpass = [float(filt) if filt not in ('NaN', 'Off', 'DC')
                             else np.Inf for filt in highpass]
-                info['highpass'] = np.min(np.array(highpass, dtype=np.float))
+                info['highpass'] = np.max(np.array(highpass, dtype=np.float))
```

Comment:
What does ``hp_s`` stand for? Here the highpass value seems to be treated differently based on it.